### DISCUSSION TITLE
Multiple on same page, all updating on drag end

### DISCUSSION THREAD

**Discussion by SteveEarl-AWS** [chrispahm/chartjs-plugin-dragdata#123] [Q&A]
I have an app in which I’m trying to use chartjs as an input mechanism for data by hour and by day. I have 7 charts and each chart has a single dataset with 24 elements (one number for each hour interval). 

What I’m seeing is that I can drag datapoint on Monday, for example, then switch to Friday and as soon as I begin dragging in Friday, the Friday chart is updated with Monday’s data. 

If I then drag in Wednesday, the Wednesday chart is updated with Monday’s and Wednesday’s cumulative changes. 

I’ve tried this in the most simplest form using example drag data examples and can reproduce the behavior. 

Has anyone seen this and solved for it?

**Comment by karloluis** [comment]
What is the chart type you are using? Doesn't seem to happen with radar chart.

**Comment by artus9033** [comment]
@SteveEarl-AWS if you could provide at least the info regarding which type of chart you are using, this could let us help you. Ideally, an MWE snippet / repo would be awesome.

There are some things I've already discovered in the code we currently have that should be changed and will be changed in the future. The current implementation uses global variables for storing the state of a plugin instance, whereas it should rather store these as properties of an instance of a class. Yet, if I were to bet (since we don't have the code nor even know the chart type / use case), I would suspect a problem in the driver code.
